advisories/hackage/process/HSEC-2024-0003.md

```diff
@@ -147,6 +150,7 @@ bump.  Because we expect very few (if any) users will be impacted by
 the behavioural change, the GHC team made a pragmatic decision to
 avoid the disruption that a major version bump would cause.
 
+A follow-up fix was released in ***process-1.6.23.0***.
```

Comment:
I think we should explain what was deficient in the initial fix.

Reply:
I added a note.

Reply:
Could you please push it? :)

Reply:
Thanks, updated commentary LGTM.